lerna.json

```diff
@@ -4,7 +4,8 @@
     "./cockpit",
     "./examples/*",
     "./odata-mock",
```

Comment:
still using app-connector-client, not api-server

Reply:
done